docs/src/getting-started/installation.md

```diff
@@ -441,3 +445,91 @@ the [source code](https://github.com/foundry-rs/starknet-foundry) as follows:
 2. Run `cd starknet-foundry && cargo build --release`. This will create a `target` directory.
 3. Move the `target` directory to the desired location (e.g. `~/.starknet-foundry`).
 4. Add `DESIRED_LOCATION/target/release/` to your `PATH`.
+
+# Uninstalling
```

Comment:
I think this could be separate file, we could nest it under `Installation` in `SUMMARY.md`

Then just add a link to installation section to it as well.